app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
-    <uses-sdk tools:overrideLibrary="androidx.wear.remote.interactions" />
+    <uses-sdk tools:overrideLibrary="androidx.wear.remote.interactions,androidx.car.app.projected" />
```

Comment:
Does this breaks the compiling of the automotive build?

Reply:
This library has a minSdk of 23, Automotive has a minSdk of 29 but the normal app 21 which is why this is required here but not for Automotive.